Mu2eKinKal/inc/KKPanelHit.hh

```diff
@@ -15,47 +16,41 @@ namespace mu2e {
     double rcell_; // straw radius
     KKStrawHitUpdater(double mindoca,double maxdoca, bool nulltime) : mindoca_(mindoca), maxdoca_(maxdoca), nulltime_(nulltime) {}
   };
+
+  using KinKal::WireHitState;
   template <class KTRAJ> class PanelHit : public KinKal::Hit<KTRAJ> {
     public:
+      static constexpr MAXNHIT=4;
       using HIT = KinKal::Hit<KTRAJ>;
-      using SHCOLL = std::vector<shared_ptr<KKWireHit<KTRAJ>>;
+      using SHCOLL = std::arra<shared_ptr<KKWireHit<KTRAJ>,MAXNHIT>;
```

Comment:
Is this a typo?  std::arra ?  The code must never instantiate this template or it would not have compiled?

Reply:
Thanks for fixing this.  I will wait for the CI tests to complete.  If they pass I will merge.